clusterman/cli/status.py

```diff
@@ -76,6 +80,7 @@ class StatusJsonObject(TypedDict):
     target_capacity: float
     non_orphan_fulfilled_capacity: float
     resource_groups: List[ResourceGroupJsonObject]
+    migrations: Collection[MigrationEvent]
```

Comment:
hmm, we should probably pick a style and be consistent - we've mixed using the typing ABCs with the concrete types all over the place (in paasta too, to be fair :p)

personally, I prefer having function params be typed using the abstract classes and everything else using the concrete ones - but i don't know if anyone else has opinions

that said, if we wanted to be consistent here: this should be 
```suggestion
    migrations: List[MigrationEvent]
```
which i think might necessitate changing the type in a couple other places too?